src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -117,7 +117,21 @@ private CloudSdk(Path sdkPath, String appCommandMetricsEnvironment,
    * @throws CloudSdkOutOfDateException when the installed Cloud SDK is too old 
    */
   public void runAppCommand(List<String> args) throws ProcessRunnerException {
-    runGcloudCommand(args, "app");
+    runGcloudCommand(args, null, "app");
```

Comment:
FYI, Eclipse doesn't have the support, and even in IntelliJ, the automatic labeling messes with the right margin guideline (i.e., the guideline at column 100 becomes meaningless), so I turn off the feature.